src/shared/components/notifications/Notifications.tsx

```diff
@@ -52,7 +52,9 @@ const Notifications: FC = () => {
               style={{maxWidth: '600px', alignItems: 'center'}}
             >
               <span style={styles}>
-                <span className="notification--message">{message}</span>
+                {message && (
+                  <span className="notification--message">{message}</span>
+                )}
```

Comment:
This is a subtle and necessary change.

When `message` is an empty string, previously the span would be rendered (due to styling) as an inline block, which means an invisible newline is inserted causing extra spacing at the top.

With this change, when `message` is an empty string, we don't render the span at all. Nothing is inserted and spacing is consistent.